web/main/models.py

```diff
@@ -2021,12 +2024,33 @@ class ContentNode(
     resource_id = models.BigIntegerField(blank=True, null=True)
 
     objects = _ContentNodeManager()
+
     tracked_fields = ["headnote"]
 
     # Stores the number of ‘read’ characters in a content
     # Length of content, less elided text and html tags.
     reading_length = models.IntegerField(null=True)
 
+    is_instructional_material = models.BooleanField(
+        default=False,
+        help_text="This content should only be made available on the frontend to verified professors, staff, or users with editing privilege",
```

Comment:
This help test is a lie; the feature is not currently available to staff/admins. It's an omission but it's a useful one since it lets us easily toggle between professor/non-professor states without having to drop our admin privilege. When it gets closer I'll have the behavior match the docs.